flexmeasures/ui/utils/color_defaults.py

```diff
@@ -0,0 +1,51 @@
+def get_color_settings(account):
+    # Initialize colors with defaults
+    primary_color = (
+        account.primary_color or account.consultancy_account.primary_color or "#1a3443"
+    )
+    secondary_color = (
+        account.secondary_color
+        or account.consultancy_account.secondary_color
```

Comment:
I ran into this error:

```
File "/home/nicolas/workspace/seita/flexmeasures/flexmeasures/ui/utils/color_defaults.py", line 4, in get_color_settings
    account.primary_color or account.consultancy_account.primary_color or "#1a3443"
AttributeError: 'NoneType' object has no attribute 'primary_color
```

I found out why - we need to protect against `consultancy_account` being `None`.

Reply:
yes i am running into the same thing on my end, ill handle that edge case

Reply:
i have handled for this edge case now